acceptance-tests/src/test/resources/blackbox/application.yml

```diff
@@ -0,0 +1,112 @@
+#SPDX-FileCopyrightText: 2017-2024 Enedis
+#SPDX-License-Identifier: Apache-2.0
+
+server:
+  instance-name: local dev
+  port: 8443
+  http:
+    port: 8080
+    interface: localhost
+  compression:
+    enabled: true
+    mime-types: text/html,text/xml,text/plain,text/css,text/javascript,application/javascript,application/json
+    min-response-size: 1024
+  ssl:
+    keyStore: 'classpath:security/https/server.jks'
```

Comment:
Where are those files ? I cannot find them (in acceptance-tests module).
Same for truststore.
Same for jasypt private key below.

Reply:
they came from local-dev packaging